selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -290,8 +290,6 @@ public void waitItem(String path) {
    * @param timeout waiting timeout in seconds
    */
   public void waitItem(String path, int timeout) {
-    loader.waitOnClosed();
```

Comment:
We need to ensure that that there is no loader in front of IDE to make the tests more stable.

Reply:
No, that makes no sense! In Pseudocode, this would read:
1) wait for condition x
2) wait for condition y.

if we never do anything with "x", we can just remove the first wait

Reply:
This method is being called from other tests which rely on `loader.waitOnClosed()`. If you are removing this wait from the method, you should add `loader.waitOnClosed()` to the callers.